toolkit/tools/imagegen/configuration/partitionflag_test.go

```diff
@@ -0,0 +1,79 @@
+// Copyright Microsoft Corporation.
+// Licensed under the MIT License.
+
+package configuration
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+//TestMain found in configuration_test.go.
+
+var (
+	validPartitionFlags = []PartitionFlag{
+		PartitionFlag("esp"),
+		PartitionFlag("grub"),
+		PartitionFlag("bios_grub"),
+		PartitionFlag("boot"),
+		PartitionFlag("dmroot"),
```

Comment:
Why not use the constants? Or define this variable inside `partitionflag.go` and use it both here and for validation there?

Reply:
This was to protect against unintended modifications to the constants.  This protects against breaking old json files unintentionally.

Reply:
Got it, make sense.

---
In reply to: [549883358](https://github.com/microsoft/CBL-Mariner/pull/479#discussion_r549883358) [](ancestors = 549883358)